client/lcd/swagger-ui/swagger.yaml

```diff
@@ -3565,4 +3565,4 @@ definitions:
       gas:
         type: number
         format: integer
-        example: 10000
\ No newline at end of file
+        example: 10000
```

Comment:
That was added by my text editor. I think ending blank line is a sane default, but feel free to remove it if not wanted.